src/wp-includes/blocks/post-excerpt.php

```diff
@@ -25,7 +25,7 @@ function render_block_core_post_excerpt( $attributes, $content, $block ) {
 	}
 
 	$more_text           = ! empty( $attributes['moreText'] ) ? '<a class="wp-block-post-excerpt__more-link" href="' . esc_url( get_the_permalink( $block->context['postId'] ) ) . '">' . wp_kses_post( $attributes['moreText'] ) . '</a>' : '';
-	$filter_excerpt_more = function( $more ) use ( $more_text ) {
+	$filter_excerpt_more = static function( $more ) use ( $more_text ) {
```

Comment:
Note that https://github.com/WordPress/gutenberg/pull/50723 had another `static` addition in this file:

https://github.com/WordPress/gutenberg/blob/e3194a1d1d9869560260a3bba0939a28a52e737f/packages/block-library/src/post-excerpt/index.php#L80-L95

I guess it will come in another backport or the change in core wasn't "for-ported" to Gutenberg?